File: src/main/frontend/src/app/common/institution.ts
export class Institution {

  constructor(private _institution: String, private _city: String, private _state: String, private _degree: String,
              private _startDate: String, private _endDate: String, private _achievements: String) {
  }

  get institution(): String {
    return this._institution;
  }

  set institution(value: String) {
    this._institution = value;
  }

  get city(): String {
    return this._city;
  }

  set city(value: String) {
    this._city = value;
  }

  get state(): String {
    return this._state;
  }

  set state(value: String) {
    this._state = value;
  }

  get degree(): String {
    return this._degree;
  }

  set degree(value: String) {
    this._degree = value;
  }

  get startDate(): String {
    return this._startDate;
  }

  set startDate(value: String) {
    this._startDate = value;
  }

  get endDate(): String {
    return this._endDate;
  }

  set endDate(value: String) {
    this._endDate = value;
  }

  get achievements(): String {
    return this._achievements;
  }

  set achievements(value: String) {
    this._achievements = value;
  }
}

File: src/main/java/org/_3rev/resume/valueobject/Institution.java
package org._3rev.resume.valueobject;

public class Institution {

    private String institution;
    private String city;
    private String state;
    private String degree;
    private String achievements;

    public Institution() {
        super();
    }

    public Institution(String institution, String city, String state, String degree, String achievements) {
        this.institution = institution;
        this.city = city;
        this.state = state;
        this.degree = degree;
        this.achievements = achievements;
    }

    public String getInstitution() {
        return institution;
    }

    public void setInstitution(String institution) {
        this.institution = institution;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getState() {
        return state;
    }

    public void setState(String state) {
        this.state = state;
    }

    public String getDegree() {
        return degree;
    }

    public void setDegree(String degree) {
        this.degree = degree;
    }

    public String getAchievements() {
        return achievements;
    }

    public void setAchievements(String achievements) {
        this.achievements = achievements;
    }
}
